### PR TITLE
Add `scale_prism`

### PR DESCRIPTION
**Pull request by damonbayer**
``` r
library(forecasttools)
ex_dat <- data.frame(x = 1:6, y = 1:6,
                     z = c("Minimal", "Low", "Moderate", "High", "Very High", NA))
# Basic plot
ex_dat |>
  ggplot2::ggplot(ggplot2::aes(x, y, color = z)) +
  ggplot2::geom_point(size = 5) +
  scale_color_prism()
```

``` r

# Dropping levels drops them from the legend
ex_dat |>
  tidyr::drop_na() |>
  dplyr::filter(z != "Moderate") |>
  ggplot2::ggplot(ggplot2::aes(x, y, color = z)) +
  ggplot2::geom_point(size = 5) +
  scale_color_prism()
```

``` r

# New level gets converted to NA
ex_dat |>
  tibble::add_row(x = 7, y = 7, z = "new level") |>
  ggplot2::ggplot(ggplot2::aes(x, y, color = z)) +
  ggplot2::geom_point(size = 5) +
  scale_color_prism()
```

<sup>Created on 2024-12-03 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

### CODE REVIEW

File: R/scale_prism.R [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#'
+#' # New level gets converted to NA
+#' ex_dat |>
+#'   tibble::add_row(x = 7, y = 7, z = "new level") |>
+#'   ggplot2::ggplot(ggplot2::aes(x, y, color = z)) +
+#'   ggplot2::geom_point(size = 5) +
+#'   scale_color_prism()
+scale_fill_prism <- function(...) {
+  ggplot2::scale_fill_manual(
+    values = prism_colors$risk_levels,
```
Comment (dylanhmorris) [inline R/scale_prism.R:55]:
```suggestion
    values = prism_colors$activity_levels,
```

File: R/scale_prism.R [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    labels = \(x) ifelse(is.na(x), prism_colors$na_label, x),
+    ...
+  )
+}
+
+#' @export
+#' @rdname scale_prism
+scale_color_prism <- function(...) {
+  ggplot2::scale_color_manual(
+    values = prism_colors$risk_levels,
```
Comment (dylanhmorris) [inline R/scale_prism.R:68]:
```suggestion
    values = prism_colors$activity_levels,
```

**Review by dylanhmorris — COMMENTED** [review]
A few minor lanugage tweaks

File: R/scale_prism.R [hunk 3]
```diff
@@ -0,0 +1,79 @@
+prism_colors <- list(
+  risk_levels = c(
+    "Minimal" = "#D7F2ED",
```
[the remark this answers is not in the document]
Comment (damonbayer) [reply R/scale_prism.R:3]: Maybe we should accept both for backwards compatibility?

**Pull request merged**